grc/blocks/variable_struct.xml.py

```diff
@@ -93,5 +93,5 @@ def make_xml(num_fields):
 
     data = make_xml(MAX_NUM_FIELDS)
 
-    with open(filename, 'w') as fp:
+    with open(filename, 'wb') as fp:
```

Comment:
That's fine. In general, we should probably request utf-8 encoding explicitly. makes no difference here, though